src/Symfony/Component/Console/Event/ConsoleExceptionEvent.php

```diff
@@ -48,9 +48,9 @@ public function getException()
      *
      * This exception will be thrown if no response is set in the event.
      *
-     * @param \Exception $exception The thrown exception
+     * @param \Exception|null $exception The thrown exception
      */
-    public function setException(\Exception $exception)
+    public function setException(\Exception $exception = null)
```

Comment:
Strictly speaking, this is a BC break. But only the public API is important for event classes, so I don't think we should consider this a BC break.